src/nvim/version.c

```diff
@@ -2751,8 +2752,15 @@ void maybe_intro_message(void)
 /// Or with the ":intro" command (for Sven :-).
 ///
 /// @param colon true for ":intro"
-void intro_message(int colon)
+void intro_message(bool colon)
 {
+  bool ext = ui_has(kUIMultigrid) || ui_has(kUIMessages);
```

Comment:
Since the intro screen now works for multigrid (if we ignore the bug that `:into` doesn't work), I think this should only apply to `ext_messages`.